server/src/services/apps.service.ts

```diff
@@ -186,12 +186,19 @@ export class AppsService {
     return await viewableAppsQb.getMany();
   }
 
-  async update(appId: string, appUpdateDto: AppUpdateDto, manager?: EntityManager) {
+  async update(appId: string, appUpdateDto: AppUpdateDto, organizationId: string, manager?: EntityManager) {
```

Comment:
can revert this changes if we adapt this
https://github.com/ToolJet/ToolJet/pull/6581